test/test_data/mysql.py

```diff
@@ -0,0 +1,18 @@
+mapping_test_data = [
+    {
+        "product": "mysql",
+        "version": "8.0.20",
+        "version_strings": [
+            r"mysql  Ver 8.0.20",
+            r"To buy MySQL Enterprise support, training, or other products, visit:",
+        ],
+    }
+]
+package_test_data = [
+    {
+        "url": "http://ports.ubuntu.com/pool/main/m/mysql-8.0/",
+        "package_name": "mysql-client-core-8.0_8.0.19-0ubuntu5_arm64.deb",
+        "product": "mysql",
+        "version": "8.0.19",
```

Comment:
The long test is failing:
```
=========================== short test summary info ============================
FAILED test/test_scanner.py::TestScanner::test_version_in_package[http://ports.ubuntu.com/pool/main/m/mysql-8.0/-mysql-client-core-8.0_8.0.19-0ubuntu5_arm64.deb-mysql-8.0.19]
====== 1 failed, 308 passed, 1 skipped, 8 warnings in 1347.06s (0:22:27) =======
```

Scrolling up through the failed log https://github.com/intel/cve-bin-tool/pull/840/checks?check_run_id=897731477, it looks like it's detecting mysql, but never finding the correct version.  Possibly an error in your version string?

For example:
```
WARNING  cve_bin_tool.VersionScanner:version_scanner.py:141 mysql was detected with version UNKNOWN in file /tmp/cve-bin-tool-sm_iwuz3/mysql-client-core-8.0_8.0.19-0ubuntu5_arm64.deb.extracted/usr/bin/mysqlcheck
```

Reply:
The master branch is failing due to some reasons.